src/main/java/frc/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.subsystems.intake;
+
+import static edu.wpi.first.units.Units.Radians;
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+
+public class IntakeConstants {
+    public static final double ANGLE_RATIO = 0;
+    public static MutableMeasure<Angle> DOWN_ANGLE = MutableMeasure.zero(Radians);
+    public static MutableMeasure<Angle> UP_ANGLE = MutableMeasure.zero(Radians);
+
+    public enum IntakePose {
+        UP(IntakeConstants.UP_ANGLE),
+        DOWN(IntakeConstants.DOWN_ANGLE);
```

Comment:
No need for separate constants, just declare the values here directly.

Reply:
done.

Reply:
8a21fe042ec15dac5f577d40cbb692c39c3aec95

Reply:
Make sure to push your changes before including commit hashes.